app/models/accounting/quickbooks/department.rb

```diff
@@ -0,0 +1,44 @@
+module Accounting
+  module Quickbooks
+
+    # Represents a QBO Department object and can create a reference object
+    # for a link to this object in a transaction or other QBO object.
```

Comment:
Thank you!

Can you add some info to this class and Customer describing what role the Customer and Department objects play in madeline? e.g. Customer == Coop, Department == Division? (Right?)